concepts/40 Angular Components/10 Getting Started/03 Add DevExtreme to an Angular CLI Application/03 Configure Stylesheets.md

```diff
@@ -24,6 +24,18 @@ Open the `angular.json` file and reference `dx.common.css` and a [predefined the
       ...
     }
 
+Then, go to the `src` folder, open the `index.html` file, and add the `dx-viewport` class to the `<body>` tag. This ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets.
```

Comment:
To avoid prepositions in succession, Can we change the second sentence to:
This ensures that theme colors and typography settings are applied to all page elements (not only to DevExtreme widgets).